modules/authentication/server-ts/access/jwt/index.js

```diff
@@ -6,37 +6,49 @@ import resolvers from './resolvers';
 import schema from './schema.graphql';
 import AccessModule from '../AccessModule';
 import settings from '../../../../../settings';
+import { MESSAGE_APPEND_CONTEXT } from '../errorMessages';
 
-const grant = async user => {
-  const refreshSecret = settings.user.secret + user.passwordHash;
-  const [accessToken, refreshToken] = await createTokens(user, settings.user.secret, refreshSecret);
+const grant = async identity => {
+  const refreshSecret = settings.auth.secret + identity.passwordHash;
```

Comment:
The Auth module doesn't know what fields `identity` consists of.

Reply:
Done.